File: BlackJack.ViewModels/PlayerViews/GetAllPlayersPlayerView.cs
using System.Collections.Generic;

namespace BlackJack.ViewModels.PlayerViews
{
    public class GetAllPlayersPlayerView
    {
        public List<PlayerGetAllPlayersPlayerViewItem> Players { get; set; }

        public GetAllPlayersPlayerView()
        {
            Players = new List<PlayerGetAllPlayersPlayerViewItem>();
        }
    }

    public class PlayerGetAllPlayersPlayerViewItem
    {
        public string PlayerId { get; set; }
        public string UserName { get; set; }
        public decimal Balance { get; set; }
        public decimal Bet { get; set; }
    }
}

File: BlackJack.WEB/ClientApp/src/app/shared/entities/player.views/get-all-players.player.view.ts
export class GetAllPlayersPlayerView {
  players?: Array<PlayerGetAllPlayersPlayerViewItem>;

  constructor() {
    this.players = new Array<PlayerGetAllPlayersPlayerViewItem>();
  }
}

export class PlayerGetAllPlayersPlayerViewItem {
  userName?: string;
  playerId?: string;
  balance?: number;
  bet?: number;
}
